theme/admin.py

```diff
@@ -52,21 +68,15 @@ def get_form(self, request, obj=None, **kwargs):
 
 class QuotaAdmin(admin.ModelAdmin):
     model = UserQuota
+    form = UserQuotaForm
 
-    list_display = ('user', 'allocated_value', 'used_value', 'data_zone_value', 'unit', 'zone')
+    list_display = ('user', 'allocated_value', 'unit', 'zone')
```

Comment:
I think you have a migration that will remove `unit` so maybe remove it here?

Reply:
Unit is still used, it is jsut saved in the minio quota system.

Reply:
got it, thanks